Formula/lux.rb

```diff
@@ -0,0 +1,37 @@
+class Lux < Formula
+  desc "Lux (LUcid eXpect scripting) simplifies test automation and provides an Expect-style execution of commands"
+  homepage "https://github.com/hawk/lux"
+  url "https://github.com/hawk/lux/archive/lux-1.19.3.tar.gz"
+  sha256 "e2dd36275c76a087a4b4577bd056a6928a5e5ba5b00d8c366e2a4f94b16f921e"
+
+  head do
```

Comment:
Simple `head "...."` would work, no need for a block

Reply:
Thanks for the feedback.